overrides/home.html

```diff
@@ -314,7 +314,7 @@ <h2>Contributions Welcome</h2>
             <p class="normal-text">Want to join the fun on Github? New users are always welcome!</p>
           </a>
         </div>
-        <div class="pane">
+        <div class="col">
```

Comment:
Is this accidental?

Reply:
Yes, I did not commit this on this branch and don't know where it is coming from. I tried rebasing and that failed too. I am getting errors could not read '.git/rebase-merge/head-name'. I have tried to continue, skip, abort, quit and all give me same error. currently, if I do log I don't see this commit but I don't know how to move this here. 
P.S: I am very new to git.